app/src/main/java/com/openclassrooms/realestatemanager/model/Estate.kt

```diff
@@ -31,6 +36,23 @@ data class Estate(
         const val ROOM_TABLE_NAME : String = "estate"
     }
 
+    fun geocodePosition(context: Context) {
+        val geocoder = Geocoder(context)
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+            geocoder.getFromLocationName("$houseNumber $street, $zipCode $city, $country, ",1) { addresses ->
+                latitude = addresses[0].latitude
+                longitude = addresses[0].longitude
+            }
+        } else {
+            @Suppress("DEPRECATION")
+            val addresses = geocoder.getFromLocationName("$houseNumber $street, $zipCode $city, $country, ",1)
```

Comment:
Est-ce que la virgule en fin de chaîne de caractères est voulue ? 

```suggestion
            val addresses = geocoder.getFromLocationName("$houseNumber $street, $zipCode $city, $country", 1)
```

Reply:
Non c'est un erreur, j'ai été trop vite. Merci